index.html

```diff
@@ -16,58 +13,251 @@
   <body>
 
   <header>
-       <nav class="navbar navbar-expand-md navbar-light">
-        <div class="container">
-        <a href="#" class="navbar-brand">My Logo</a>
-        <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbarCollapse">
-            <span class="navbar-toggler-icon"></span>
-        </button>
-
-        <div class="collapse navbar-collapse justify-content-between" id="navbarCollapse">
-            <div class="navbar-nav"> </div>
-            <div class="navbar-nav">
-                <a href="#" class="nav-item nav-link active">Porfolio</a>
-                <a href="#" class="nav-item nav-link">About</a>
-                <a href="#" class="nav-item nav-link">Contact</a>
-            </div>
-        </div>
-        </div>
+    <nav class="navbar">
+             <div class="logo">
+                 <a href="#"> My Logo </a>
+             </div>
+
+                 <ul class="nav-list">
+                      <li class="list-item">
+                         <a href="#"> Portfolio </a>
+                      </li>
+                     <li class="list-item">
+                         <a href="#"> About </a>
+                      </li>
+                      <li class="list-item">
+                         <a href="#"> Contact </a>
+                      </li>
+                 </ul>
+             <div class="menu">
+                  <div class="menu-line"></div>
+                  <div class="menu-line"></div>
+                  <div class="menu-line"></div>
```

Comment:
- [ ] thes should be enclosed it on nav section

Reply:
I think, this is already in nav section or I don't really understand what you m

![Screenshots](https://user-images.githubusercontent.com/6615394/126033887-c37c3517-605b-4773-8120-4deb3df31cd6.PNG)

Reply:
Well resolved